product_docs/docs/edbcloud/beta/using_cluster/connecting_your_cluster/02_connect_on_premises.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: Connecting from Networks Outside Azure
+---
+
+
+If your applications are deployed on your on-premises environments, or on other Cloud providers instead Azure, you can set up a secure tunnel between your on premises network and the Azure virtual network. 
+
+There are a few options you can use.
+
+## Connecting Using Azure VPN Gateway (Point-to-Site)
+
+Azure VPN gateway provides you an encrypted tunnel to communicate
+with your IT infrastructures on a Azure Private Network over complex
+network environments like the internet.
+
+Point-to-Site VPN connections are useful when you want to connect to
+your EDB Cloud cluster from a remote location, such as when you do
+administrative operations from home. You can also use P2S instead of a
+Site-to-Site VPN when you have only a few clients that need to connect
```

Comment:
We should talk about the benefits of P2S here, not just gesture towards the limitations. /cc @zozowell